data/id/legislature/Alison-Ali-Rabe-29146ec1-cd85-48fb-9ffa-364299c26c47.yml

```diff
@@ -0,0 +1,22 @@
+id: ocd-person/29146ec1-cd85-48fb-9ffa-364299c26c47
+name: "Alison \u201CAli\u201D Rabe"
+email: ARabe@senate.idaho.gov
+image: https://legislature.idaho.gov/wp-content/uploads/sessioninfo/2023/directory/Rabe6687.jpg
```

Comment:
@bfossen-ce this person was in senate district 17 from 12/01/2020 through 12/01/2022